vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp

```diff
@@ -41,7 +41,7 @@ class ScrollViewShadowNode : public ShadowNodeBase {
   void EmitScrollEvent(
       const winrt::ScrollViewer &scrollViewer,
       int64_t tag,
-      winrt::hstring &&eventName,
+      const char *eventName,
```

Comment:
Most of the changes in this file revert @NickGerleman's changes for a winrt::hstring based BatchingEventEmitter